src/panels/NightSkyPanel/tabs/NightSkyMarkingsTab.tsx

```diff
@@ -0,0 +1,147 @@
+import { Button, Grid, Text } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+
+import { IconLabelButton } from '../components/IconLabelButton';
+
+export function NightSkyMarkingsTab() {
+  const openspace = useOpenSpaceApi();
+
+  return (
+    <>
+      <Button
+        mb={'lg'}
+        fullWidth
+        onClick={() => openspace?.action.triggerAction('os.nightsky.HideAllMarkings')}
+      >
+        HIDE ALL
+      </Button>
+      <Text my={'md'}>Alt/Az</Text>
```

Comment:
These should be titles instead of sized text, to provide hierarchy for screen reader users. If you don't like the look of them, you can change the font weight

E.g. this looks similar to what you have now
```suggestion
      <Title order={3} my={'sm'} fw={400}>
        Alt/Az
      </Title>
```
But unsure if the order should be 2 or 3 here. I think that the tab title should probably count as the `h2` (need to fix that this is the case, though), so this is `h3`

Do the same for all other texts in this tab